docs/develop/guides/working-with-momento-tokens.md

```diff
@@ -0,0 +1,131 @@
+---
+sidebar_position: 4
+sidebar_label: Momento Authentication
+title: Momento Authentication With Expiring Tokens
+description: Learn how to use expiring tokens to enhance the security of your application
+pagination_next: null
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Momento Authentication With Expiring Tokens
+
+To access Momento from your application, you need a Momento Token. If you wish to use a token that never expires, or you are just testing out Momento in a development environment, you may stop reading now. If you wish to enhance the security of your application by using expiring tokens that need to be rotated periodically, we will walk through how to do exactly that in this article.
+
+# Generating Expiring Tokens
+
+## Step 1: Sign up or log into the Momento Console
+Go to the Momento Console and follow the instructions to login with your email address, Google account, or GitHub account.
+
+![image](/img/getting-started/console.png)
```

Comment:
@NoSQLKnowHow I don't think these image links (copied from the Getting Started page) are working due to the relative links being broken in this nested file. Could you help fix these? I literally just want to copy the images over. Thanks!